agent/ecscni/types.go

```diff
@@ -113,7 +113,7 @@ type ENIConfig struct {
 	// IPV4Address is the ipv4 of eni
 	IPV4Address string `json:"ipv4-address"`
 	// IPV6Address is the ipv6 of eni
-	IPV6Address string `json:"ipv6-address, omitempty"`
```

Comment:
`agent/ecscni/types.go:116: struct field tag `json:"ipv6-address, omitempty"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value`